app/__init__.py

```diff
@@ -19,6 +20,11 @@
 login_manager = LoginManager()
 login_manager.session_protection = 'strong'
 login_manager.login_view = 'login.index'
+csrf = CSRFProtect()
```

Comment:
Seems it's not fully functional and there is  an error when trying to add an item to the todo list.

`flask_wtf.csrf.CSRFError
flask_wtf.csrf.CSRFError: 400 Bad Request: The CSRF token is missing.`